docs/source/happy.rst

```diff
@@ -300,6 +300,25 @@ Or exclude matching tests from being run:
    $ bin/test -t \!robot
 
 
+Running tests with the Google Chrome browser
+--------------------------------------------
+
+In our setup, robot framework uses Firefox to run the robot tests.
```

Comment:
It's not name of executable, but a fixed set of browser names supported by selenium and selenium2library. Check Open browser -keyword in selenium2library for available options.

ROBOT_BROWSER should work. I've been using it all the time with phantomjs. The value gets passed as BROWSER variable in robot tests and used by our Open test browser -keyword.


On 23. kesäkuuta 2016 klo 13.27 +0300, Johannes Raggam<notifications@github.com>, wrote:
>  
> Indocs/source/happy.rst(https://github.com/plone/plone.app.robotframework/pull/56#discussion_r68211084):
>  
> >@@ -300,6 +300,25 @@ Or exclude matching tests from being run:>$ bin/test -t \!robot>>>+Running tests with the Google Chrome browser>+-------------------------------------------->+>+In our setup, robot framework uses Firefox to run the robot tests.
>  
> Is that true? I tried to temporarily set Chrome instead of Firefox as default browser, but still Firefox was started. That brought me to the conclusion, that Firefox might be set in config.
>  
>  
> —
> You are receiving this because you were mentioned.
> Reply to this email directly,view it on GitHub(https://github.com/plone/plone.app.robotframework/pull/56/files/2db142dae1f46f6ff42dc6ea0babec82100051b8#r68211084), ormute the thread(https://github.com/notifications/unsubscribe/AAJyv06UlAZ9zRGdO8hORehSgFsMVqjVks5qOl9-gaJpZM4I8qi9).
>  
>  
>